### ISSUE TITLE
New field for article number

### ISSUE THREAD

**Issue by cpmpercussion** [NIME-conference/NIME-bibliography#32]
The 2020 proceedings are ordered by article number (and also have page numbers) which I think is neat.

I propose adding an article number to the previous proceedings, this might be useful in some proceedings where there are no page numbers (editors hate this), and also help us keep track of the ordering of papers within an edition of the proc automatically (e.g., for generating a page with all the papers from a certain year in page order).

The article number would simply start at **1** for whatever paper had page 1 and go from there, restarting for each year.

There are two commonly used fields, but both a bit nonstandard:

`articleno`: I've seen this one fairly commonly in some ACM DL proceedings -- I guess this is my preference as it is kind self-explanatory as to what it means.

`eid`: Haven't see this one so much, but could work too. Here's [jabref's description](https://docs.jabref.org/advanced/fields#non-standard-fields).

What do you think?

**Comment by cpmpercussion** [comment]
bit of a discussion on these two options here: https://github.com/borisveytsman/acmart/issues/133

**Comment by alexarje** [comment]
Yes, I agree that this would be useful. I think `articleno` is easier to understand. One challenge is that for most years, the numbering is based on the submission number and not the proceedings order. 

Also, the proceedings order varies a lot. Sometimes the papers were organised after the submission number, other times after some alphabetical logic, and other times again after when they were presented in the conference. I guess this doesn't matter too much, though, as long as we are consistent.
